### PR TITLE
Style bookstore

### PR DESCRIPTION
**Pull request by getssh**
- Add style for components
- Followed the given sample design 
- used Flexbox for positioning
- checked linter errors

### CODE REVIEW

File: src/components/header/Header.css [hunk 1]
```diff
@@ -1,3 +1,84 @@
...
+}
+
+a {
+  color: #121212;
+}
+
+a:active {
+  color: #121212;
+}
+
```
Comment (brainconnect93) [inline src/components/header/Header.css]: - [x] Hi @getssh, Great effort working on the styling of the bookstore project, But currently i would suggest you work on some alignment in your project. The `Bookstore CMS` logo and the `icon` align with the books and form vertically from top to bottom as seen in the Zeplin Design. 

Below is the difference spotted 👇 

| Your Design 👇 | Zeplin Design 👇 |
| ------------- | ------------- |
| tam | org |

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @getssh, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

-  Descriptive Pull Request title and summary. ✔️  
- Styling of the bookstore CMS project ✔️ 
- Zeplin Design template used for the design ✔️ 
- There is good use of Git flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/header/Header.css (hunk 1 above)
Comment (getssh) [reply src/components/header/Header.css]: Hello @brainconnect93, Thank you for the review. I have implemented the requested change

File: src/components/header/Header.css [hunk 2]
```diff
@@ -1,3 +1,84 @@
...
+.bookstore-header {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  width: 100%;
+  height: 5.938rem;
+  margin: 0;
+  padding: 1.438rem 9.088rem 1.688rem 10.75rem;
+  background-color: #fff;
+}
```
Comment (OyePriscilla) [inline src/components/header/Header.css]: - [x] As pointed out earlier, can you please ensure that the contents of the header (`Bookstore-cms` and the `logo`) is well aligned with the other elements according to the given template. 👍 Kindly don't be discouraged, reproducing perfectly a given design shows how well we pay attention to details as developers; and I'm sure recruiters are always on look out for such. ℹ️ 

## Your design
Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @getssh 👏 👏 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Add styling to app ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/header/Header.css (hunk 2 above)
Comment (getssh) [reply src/components/header/Header.css]: Hey @OyePriscilla, Thank you again for the review. I make changes according to the requested change. Thanks

**Review by algerina — APPROVED** [review]
Hi  @getssh ,

Great job so far! 

May I highlight the following: 
- [x] Your project looks awesomel✔️ 
- [x] Well done implementing all the requirements✔️ 

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**